### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by terHahaha**

### CODE REVIEW

**Comment by vandenman** [comment]
@terHahaha For some reason I can only assign one reviewer. Could you check under settings if you can change this somewhere?
@Dom-Bach please also review this PR.

**Comment by terHahaha** [comment]
> @terHahaha For some reason I can only assign one reviewer. Could you check under settings if you can change this somewhere?
> @Dom-Bach please also review this PR.


@vandenman It's not letting me change my settings, I don't understand why.
@erenasena  Didn't mean to remove you. Tried something out.

File: week1 [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+"""
+'''
+Sprouts
+Sprouts is a simple 3 rule game created by John Horton Conway and Mike Paterson. 
+The game begins with some initial number of dots greater than 2 and players take turns connecting dots with lines. 
+Each time a player draws a line, that player places a dot on that line. 
+This implementation of the game allows users to play sprouts in ascii from the command line.
+
+The Rules
+1.The must not touch or cross itself or any other line.
```
Comment (erenasena) [inline week1:16]: The line?

**Review by erenasena — COMMENTED** [review]
You seem to have a good idea of how to proceed. I could comment that the functions should be specified in more detail but I think that would be a bit superfluous on my part. You will probably do that next week. Although the first function to run the game really could have been specified better. I am not sure what that means.

File: week1 [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+2.The new spot cannot be placed on top of one of the endpoints of the new line. 
+Thus the new spot splits the line into two shorter lines.
+3. No spot may have more than three lines attached to it. 
+For the purposes of this rule, a line from the spot to itself counts as two attached lines and new spots are counted as having two lines already attached to them.
+
+Implementation
+I am using Python for this project, as I think it will allow for more creative freedom and flexibility. I will start by implementing a proof of principle that works and add upon this.
+
+Packages (still deciding on the best ones):
+Pygame: used for design aspects of the game
```
Comment (Dom-Bach) [inline week1:26]: _I'm not sure whether the purpose of the readme is to explain the code to others or only to be an overview with yourself. If it's the latter, you can disregard most of my comments (as long as you know what you meant)._

"Design" is ambiguous. Do you mean its looks, its controls, its rules? All these fall under game design.

File: week1 [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+Function to draw the (first) dot
+Function to recognise mouse and draw line
+Function to recognise/implement rules
+
+Must have time:
+Placement of dots
+Drawing of lines
+Implementation of rules
+Functional game 
+Scoreboard
```
Comment (Dom-Bach) [inline week1:40]: _I'm not sure whether the purpose of the readme is to explain the code to others or only to be an overview with yourself. If it's the latter, you can disregard most of my comments (as long as you know what you meant)._

As in allowing people to play best-of-X (e.g., best-of-3 is first to win two games), or what do you mean by that?
Or in global leaderboards (e.g., number of games someone wins in a row)? Then you'd need something to identify players with.

File: week1 [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Implementation
+I am using Python for this project, as I think it will allow for more creative freedom and flexibility. I will start by implementing a proof of principle that works and add upon this.
+
+Packages (still deciding on the best ones):
+Pygame: used for design aspects of the game
+Sys: used to exit/quit the game as well as other implementations
+
+Functions/Packages:
+Function to run the game
+Function to draw the (first) dot
```
Comment (Dom-Bach) [inline week1:31]: _I'm not sure whether the purpose of the readme is to explain the code to others or only to be an overview with yourself. If it's the latter, you can disregard most of my comments (as long as you know what you meant)._

So if there is only one function, you let players decide on the placement of the first dots? (If so, how? Does the first player set the first dot and the second player the second one?)

File: week1 [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May 13 19:38:56 2020
+
+@author: anne
+"""
+'''
+Sprouts
+Sprouts is a simple 3 rule game created by John Horton Conway and Mike Paterson. 
+The game begins with some initial number of dots greater than 2 and players take turns connecting dots with lines.
```
Comment (Dom-Bach) [inline week1:11]: Based on what you write later, this should read "greater than 1"

**Review by Dom-Bach — COMMENTED** [review]
Seems like you distilled the most important elements that you'll need for the finished project and that you set yourself realistic goals. Good job on that! 
I also like how you made multiple tiers of importance, all your (possible) extensions make sense and are separate enough to be implemented retrospectively (i.e., creating code for one more starting dot should be doable without adversely influencing the code that you wrote before).

I agree with Eren on his comment about "function to run the game". The fact that we don't know what it would mean in practice to us missing a bit of detail on what your "grand vision" is for this project (else we'd know what "running the game" means to you). 
For example, how does one start a game? Are all details written out in lines of code, _a la_ "sprout(NStartingDots = 3, BestOfX = 5, TurnLengthSeconds = 30, RandomOpponent = True)" or do you just write an initial command (e.g., "sprouts()") and then come to an interface/GUI to specify details?

Good luck moving forward!

**Pull request merged**